sdk/avalon_sdk/connector/direct/jrpc/jrpc_worker_registry.py

```diff
@@ -179,13 +214,20 @@ def worker_register(self, worker_id, worker_type, org_id,
         Returns:
         JRPC response with worker registry status.
         """
+
+        code, error_response = self.workerType_validation(
+                                    worker_type, json_rpc_request)
```

Comment:
Am I missing something here? I see `json_rpc_request` defined below. But its used before it's defined.  Can you please run existing/new unit test cases?

Reply:
Thanks Manoj, Since the client cannnot register worker, this function Worker Register was missed to test.
Fixed the code .